### ISSUE TITLE
[BUG]: Laser logic component does not output anything when pointed on a vehicle

### ISSUE THREAD

**Issue by Imjustjokingwithya** [typical-developers/oaklands-feedback#448]
### Roblox Username

tupisupi

### What version did this issue occur in?

1.54.6

### Describe the issue in detail.

Title and images, pretty self explanatory.

### Provide some images / videos of the issue in action.

![kuva](https://github.com/typical-developers/oaklands-feedback/assets/96913351/114b059b-cfc2-4dd8-87e4-d0e30a2913c5)
![kuva](https://github.com/typical-developers/oaklands-feedback/assets/96913351/10ea1c96-9307-4902-a5f6-67e97886e404)

### Before Submitting

- [X] I have confirmed that nobody else reported this issue.
- [X] The details of this report are legitimate.

**Comment by Silbersix** [comment]
+1 please implement this functionality again

**Comment by AmericanExpatriate** [comment]
i don't think this is a bug as the laser wasn't meant to detect structures, but it would be a good suggestion.
